job-templates/kubernetes_release.json

```diff
@@ -0,0 +1,85 @@
+{
+  "apiVersion": "vlabs",
+  "properties": {
+    "orchestratorProfile": {
+      "orchestratorType": "Kubernetes",
+      "orchestratorRelease": "1.13"
+    },
+    "masterProfile": {
+      "count": 1,
+      "dnsPrefix": "",
+      "vmSize": "Standard_D2_v3",
+      "distro": "ubuntu",
+      "extensions": [
+          {
+              "name": "win-e2e-master-extension"
+          }
+      ]
+    },
+    "agentPoolProfiles": [
+      {
+        "name": "windowspool2",
+        "count": 2,
+        "vmSize": "Standard_D2_v3",
+        "availabilityProfile": "AvailabilitySet",
+        "osType": "Windows",
+        "preProvisionExtension":
+            {
+                "name": "node_setup",
+                "singleOrAll": "all"
+            },
+        "extensions": [
+            {
+                "name": "winrm"
+            },
+            {
+              "name": "windows-patches"
+            }
+        ]
+      }
+    ],
+    "windowsProfile": {
+      "adminUsername": "azureuser",
+      "adminPassword": "replacepassword1234$"
+    },
+    "linuxProfile": {
+      "adminUsername": "azureuser",
+      "ssh": {
+        "publicKeys": [
+          {
+            "keyData": ""
+          }
+        ]
+      }
+    },
+    "servicePrincipalProfile": {
+      "clientId": "",
+      "secret": ""
+    },
+    "extensionProfiles": [
+       {
+          "name":    "node_setup",
+          "version": "v1",
+          "rootURL": "https://k8swin.blob.core.windows.net/k8s-windows/preprovision_extensions/",
+          "script":  "node_setup.ps1"
+        },
+        {
+          "name":"winrm",
+          "version": "v1"
+        },
+        {
+          "name":                "win-e2e-master-extension",
+          "version":             "v1",
+          "extensionParameters": "parameters",
+          "rootURL":             "https://k8swin.blob.core.windows.net/k8s-windows/extensions/",
+          "script":              "win-e2e-master-extension.sh"
```

Comment:
@adelina-t and I talked about this today - these will move into the repo too in another PR